modules/javafx.graphics/src/main/java/com/sun/javafx/iio/common/ImageTools.java

```diff
@@ -47,6 +49,11 @@ public class ImageTools {
      */
     public static final int PROGRESS_INTERVAL = 5;
 
+    /**
+     * Regex pattern for hasScaledName
+     */
+    private static final Pattern SCALED_FILE_PATTERN = Pattern.compile(".*@[1-9][0-9]?x(\\.[^\\.]+)?");
```

Comment:
You can use a non-capturing group instead of a capturing group if you don't intend to retrieve its value:
```suggestion
    private static final Pattern SCALED_FILE_PATTERN = Pattern.compile(".*@[1-9][0-9]?x(?:\\.[^\\.]+)?");
```